_specs/ecip-1040.md

```diff
@@ -0,0 +1,239 @@
+---
+lang: en
+ecip: 1040
+title: Generalized Account Versioning Scheme
+author: Wei Tang (@sorpaas)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/169
+status: Draft
+type: Standards Track
+category: Core
+created: 2017-12-30
```

Comment:
Should this be changed to today's date instead or when you created the specific document as an ECIP with Apache License?

Reply:
I don’t see why that’s needed. There’s technically no EIPs for those two extensions as they are never merged.

Reply:
Alrighty, just wanted to point it out. Resolving.